doc/dftb+/manual/dftbp.tex

```diff
@@ -1090,6 +1090,7 @@ \section{Hamiltonian}
   \kw{RangeSeparated} & p &  & \is{None} & \pref{sec:dftbp.RangeSep} \\
   \kw{HubbardDerivs} & p & ThirdOrder(Full) = Yes  & - & \\
   \kw{OnSiteCorrection} & p & SCC = Yes & - & \pref{sec:dftbp.Onsites} \\
+  \kw{Solvation} & m & & - & \pref{sec:dftbp.Solvation}\\
```

Comment:
Seems to be a broken reference at compile time, probably change in capitalisation for 'Solvation' in the \label{}